### DISCUSSION TITLE
Is there a way to verify Livewire is working?

### DISCUSSION THREAD

**Discussion by stevepurcell** [livewire/livewire#3387] [1. Help]
I have a project with a couple of Livewire components.  One of them works fine, and the other does not.  

The one that works fine shows the component in the Chrome Livewire debug tool, where the second does not.

Is there a way to display all of the components that Livewire / Laravel knows about?  I am accessing my new component the same way as the other one (that works).

**Comment by joshhanley** [comment]
@stevepurcell if the second component was loading properly, it should show up in the debug tool.

Can you share your component class and view for the one not working, as well as how you are adding it into your app (either how you include it in a view or your route mapping if you a binding directly to the component)?

**Comment by stevepurcell** [comment]


**Reply by joshhanley** [reply]
@stevepurcell can you please edit your comment and fix the code blocks as github is rendering it all weird. Try putting your code in triple back ticks instead of single.

Like
\```php
// Php code here
\```

\```blade
// Blade/html code here
\```
